frontend/src/i18n/locales/de.yaml

```diff
@@ -369,27 +369,32 @@ manage:
       actions:
         title: Berechtigung
         read: Lesen
-        read-description_one: Kann das Video anschauen und Metadaten sehen.
-        read-description_other: Können das Video anschauen und Metadaten sehen.
         write: Lesen und schreiben
-        write-description_one: >
+        moderate: Moderation
+        unknown: Unbekanntes Berechtigungsschema
+        admin: Seitenadmin
+        unknown-description: Dieses Berechtigungsschema ist nicht bekannt.
+        video-read-description_one: Kann das Video anschauen und Metadaten sehen.
+        video-read-description_other: Können das Video anschauen und Metadaten sehen.
+        video-write-description_one: >
           Kann außerdem das Video editieren, löschen und Metadaten sowie andere Videodetails verändern.
-        write-description_other: >
+        video-write-description_other: >
           Können außerdem das Video editieren, löschen und Metadaten sowie andere Videodetails verändern.
-        moderate: Moderation
-        moderate-description_one: >
+        series-read-description: Nutzbar als Vorlage für Videos dieser Serie.
+        series-write-description_one: >
+          Kann Serie löschen und deren Inhalt sowie Metadaten verändern.
+        series-write-description_other: >
+          Können Serie löschen und deren Inhalt sowie Metadaten verändern.
```

Comment:
I think I would explicitly mention "add to series"... "Kann Serie löschen, Metadaten ändern und Videos zur Serie hinzufügen" or sth like that?